R/bfast-deprecated.R

```diff
@@ -0,0 +1,8 @@
+#' Deprecated functions in package \pkg{bfast}.
+#' 
+#' The functions listed below are deprecated and will be removed in the future.
+#' Please use the alternative functions as listed below.
+#' Help pages for deprecated functions are available via \code{help("<function>-deprecated")}.
+#' @name bfast-deprecated
+#' @keywords internal
+NULL
```

Comment:
More of a curiosity, I'm sure this was intentional, but why is ```NULL``` necessary here?

Reply:
This is for Roxygen2, comments can't just hang in the air, they must always precede something.